main_project/controller/enrollment.py

```diff
@@ -0,0 +1,27 @@
+from sqlalchemy import text
+from main_project.model.base import cnx
+from main_project.model.enroll_and_exam import Enrollments
+
+#Taking input from user and enrolling student to course
+def student_enrollment_course():
+    inp_student_id = int(input('Enter student id:'))
+    inp_course_id = int(input('Enter course id:'))
+    sql = text('insert into enrollment (enroll_course_id,student_id) select course.course_id, student.id from student inner join course on student.student_level = course.level where not exists (select enroll_course_id,student_id from enrollment where enroll_course_id = %s and student_id= %s)  and  course.course_id = %s and student.id = %s;' % (inp_course_id,inp_student_id,inp_course_id,inp_student_id))
```

Comment:
Lets make an query.py file and store all the queries over there and import where required